src/tactic/ui/startup/security_wdg.py

```diff
@@ -412,18 +434,60 @@ def execute(my):
             # make sure the group doesn't already exist
             #login_group = LoginGroup.get_by_login_group(new_group)
 
+            project_included = False
+            user_defined_project_code = ""
+            original_new_group = new_group
+            is_project_specific = False
+            if my.kwargs.get("checked") in [True, "true", "True"]:
+                is_project_specific = True
+
             if new_group.find("/") == -1:
                 title = new_group
                 new_group = "%s/%s" % (project_code, new_group)
             else:
                 parts = new_group.split("/")
                 title = parts[1]
+                user_defined_project_code = parts[0]
+                project_included = True
 
             new_group = Common.get_filesystem_name(new_group)
 
             group = SearchType.create("sthpw/login_group")
-            group.set_value("login_group", new_group)
-            group.set_value("project_code", project_code)
+
+            # the following sections contains the 4 cases possible when creating a new group
+            # through the popup
+            
```

Comment:
All 4 lines of comments below are wrong.. It's not "the user" , It's "the group"